### ISSUE TITLE
SuiteSparse additional module paths

### ISSUE THREAD

**Issue by stachon** [hpcugent/easybuild-easyblocks#882]
To compile FreeFem++ with SuiteSparse, I needed to add additional module paths. Not sure if this should go into easyblock or easyconfig, this is what I have in easyconfig as a workaround for now:

```python
modextrapaths = {
  'LD_LIBRARY_PATH': ['SuiteSparse_config'],
  'CPATH': ['UMFPACK/include', 'AMD/include', 'SuiteSparse_config'],
  'LIBRARY_PATH': ['AMD/lib', 'BTF/lib', 'CAMD/lib', 'CCOLAMD/lib', 'CHOLAMD/lib', 'CHOLMOD/lib', 'COLAMD/lib/', 'CSparse/lib', 'CXSparse/lib', 'KLU/lib', 'LDL/lib', 'RBio/lib', 'UMFPACK/lib', 'SuiteSparse_config']
}
```

**Comment by boegel** [comment]
@stachon since there is a custom easyblock specific to SuiteSparse, it should be fixed there, in the `make_module_req_guess` method that's already implemented there

Are you up for tackling that?

**Comment by ocaisa** [comment]
For me the easyblock seems to be broken, it's not adding `lib` to `LD_LIBRARY_PATH`, only to `LIBRARY_PATH`, I think you are suffering from the same problem.

**Comment by ocaisa** [comment]
#885 solved configuration problems I had with Octave, I think the suitesparse easyconfig may need a bot of an overhaul though

**Comment by ocaisa** [comment]
@stachon Can you try the easyblock from #885 and see if it solves your problem, I think it will (unless you want a static build)

**Comment by boegel** [comment]
The intention is now to fix this via #863

**Comment by boegel** [comment]
now fixed in #863, should get merged soon
